openedx/adg/lms/webinars/admin.py

```diff
@@ -247,6 +248,8 @@ def has_change_permission(self, request, obj=None):
         return False
 
     def has_delete_permission(self, request, obj=None):
+        if request.user.groups.filter(name=WEBINAR_REGISTRATION_DELETE_PERMISSION_GROUP):
```

Comment:
Can we change this expression to `        return True if request.user.groups.filter(name=WEBINAR_REGISTRATION_DELETE_PERMISSION_GROUP) else False`